Documentation/UseCases/Index.rst

```diff
@@ -272,3 +272,49 @@ like `https://example.com/downloads/detail/some-download/hz6dFgz9/`, where
 
 .. _custom dimension: https://matomo.org/docs/custom-dimensions/
 .. _setting a custom URL: https://matomo.org/faq/how-to/how-do-i-set-a-custom-url-using-the-matomo-javascript-tracker/
+
```

Comment:
Please move the chapter before the links. The links should always be at the end of the document.

Reply:
Done